tensornetwork/contractors/opt_einsum_paths/path_contractors.py

```diff
@@ -16,15 +16,77 @@
 
 import functools
 import opt_einsum
+import tensornetwork as tn
```

Comment:
Don't import the base tensornetwork module like because you'll get a cyclic import. Prefer to just import what you need